CHANGELOG.md

```diff
@@ -12,9 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a parser and writer for the `pileup` sequence alignment format (#329)
 - Added statistics to the `synthesis/codon` package (keeping track of the observed start codon occurrences in a translation table) (#350)
 - Added option to fragmenter to fragment with only certain overhangs (#387)
-
-
-
+- GoldenGates with methylated sequences using lowercase letters (#391)
```

Comment:
include note on function signature including what I think is a methylation flag in the clone function @Koeng101?

Reply:
what should I add to the following documentation?

https://github.com/TimothyStiles/poly/blob/debc0604d0696987c634c3cceff9876d581b4cdc/clone/clone.go#L345-L347